dist/profile/manifests/jenkinscontroller.pp

```diff
@@ -335,10 +335,17 @@
     require => File[$docroot],
   }
 
+  $ci_fqdn_x_forwarded_host = "
+RequestHeader set X-Forwarded-Host \"${ci_fqdn}\"
```

Comment:
Sounds like you might want to define a template, because the amount of `if` makes this really hard to read and maintain (ref. https://puppet.com/docs/puppet/6/lang_template_erb.html#erb_use-evaluating-ERB-template-strings) .

Reply:
As discussed together and as I don't see how to pass a value to the inline_template function, I'll keep this construction.